### ISSUE TITLE
gfycat issue, SSL not trusted u wot

### ISSUE THREAD

**Issue by staplerx300** [ShareX/ShareX#1074]
TL;DR
I can upload gifs/webm manually to gfycat. Can't upload via sharex.

Here's the error, similar to previous gfycat issues i think:

System.ArgumentNullException: Value cannot be null.
Parameter name: value
   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
   at ShareX.UploadersLib.FileUploaders.GfycatUploader.TranscodeFile(String key, UploadResult result)
   at ShareX.UploadersLib.FileUploaders.GfycatUploader.Upload(Stream stream, String fileName)
   at ShareX.WorkerTask.UploadFile(Stream stream, String fileName)
   at ShareX.WorkerTask.DoUpload(Int32 retry)

Message:
The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel.

StackTrace:
   at System.Net.HttpWebRequest.GetResponse()
   at ShareX.UploadersLib.Uploader.GetResponse(HttpMethod method, String url, Dictionary`2 arguments, NameValueCollection headers, CookieCollection cookies, Stream dataStream)

**Comment by staplerx300** [comment]
pls respond

**Comment by Jaex** [comment]
Gfycat works for me. SSL errors never to do with ShareX and not to mention it is written by Microsoft in .NET library.

**Comment by staplerx300** [comment]
I uninstalled and reinstalled and it works. Not sure why that is. It was having errors with the other services to upload webm's as well though not just gfycat after I tested more. Don't have the errors saved from the other attempts though my bad :( I guess you can close this issue since it works again, i'll repost with the error if it happens again.

**Comment by staplerx300** [comment]
Also... " not to mention it is written by Microsoft in .NET library."

lol :+1:
